---a/net/minecraft/entity/ai/EntityMinecartMobSpawner.java
+++b/net/minecraft/entity/ai/EntityMinecartMobSpawner.java
@@ -7,72 +7,56 @@
 import net.minecraft.tileentity.MobSpawnerBaseLogic;
 import net.minecraft.util.BlockPos;
 import net.minecraft.world.World;
-import net.minecraftforge.fml.relauncher.Side;
-import net.minecraftforge.fml.relauncher.SideOnly;
 
-public class EntityMinecartMobSpawner extends EntityMinecart
-{
-    private final MobSpawnerBaseLogic mobSpawnerLogic = new MobSpawnerBaseLogic()
-    {
+
+public class EntityMinecartMobSpawner extends EntityMinecart {
+
+    private final MobSpawnerBaseLogic mobSpawnerLogic = new MobSpawnerBaseLogic() {
+
         private static final String __OBFID = "CL_00001679";
-        public void func_98267_a(int p_98267_1_)
-        {
-            EntityMinecartMobSpawner.this.worldObj.setEntityState(EntityMinecartMobSpawner.this, (byte)p_98267_1_);
+
+        public void func_98267_a(int p_98267_1_) {
+            EntityMinecartMobSpawner.this.worldObj.setEntityState(EntityMinecartMobSpawner.this, (byte) p_98267_1_);
         }
-        public World getSpawnerWorld()
-        {
+
+        public World getSpawnerWorld() {
             return EntityMinecartMobSpawner.this.worldObj;
         }
-        public BlockPos func_177221_b()
-        {
+
+        public BlockPos func_177221_b() {
             return new BlockPos(EntityMinecartMobSpawner.this);
         }
     };
     private static final String __OBFID = "CL_00001678";
 
-    public EntityMinecartMobSpawner(World worldIn)
-    {
+    public EntityMinecartMobSpawner(World worldIn) {
         super(worldIn);
     }
 
-    public EntityMinecartMobSpawner(World worldIn, double p_i1726_2_, double p_i1726_4_, double p_i1726_6_)
-    {
+    public EntityMinecartMobSpawner(World worldIn, double p_i1726_2_, double p_i1726_4_, double p_i1726_6_) {
         super(worldIn, p_i1726_2_, p_i1726_4_, p_i1726_6_);
     }
 
-    public EntityMinecart.EnumMinecartType getMinecartType()
-    {
+    public EntityMinecart.EnumMinecartType getMinecartType() {
         return EntityMinecart.EnumMinecartType.SPAWNER;
     }
 
-    public IBlockState getDefaultDisplayTile()
-    {
+    public IBlockState getDefaultDisplayTile() {
         return Blocks.mob_spawner.getDefaultState();
     }
 
-    protected void readEntityFromNBT(NBTTagCompound tagCompund)
-    {
+    protected void readEntityFromNBT(NBTTagCompound tagCompund) {
         super.readEntityFromNBT(tagCompund);
         this.mobSpawnerLogic.readFromNBT(tagCompund);
     }
 
-    protected void writeEntityToNBT(NBTTagCompound tagCompound)
-    {
+    protected void writeEntityToNBT(NBTTagCompound tagCompound) {
         super.writeEntityToNBT(tagCompound);
         this.mobSpawnerLogic.writeToNBT(tagCompound);
     }
 
-    @SideOnly(Side.CLIENT)
-    public void handleHealthUpdate(byte p_70103_1_)
-    {
-        this.mobSpawnerLogic.setDelayToMin(p_70103_1_);
-    }
-
-    public void onUpdate()
-    {
+    public void onUpdate() {
         super.onUpdate();
         this.mobSpawnerLogic.updateSpawner();
     }
 
-    @SideOnly(Side.CLIENT)
-    public MobSpawnerBaseLogic func_98039_d()
-    {
-        return this.mobSpawnerLogic;
-    }
-}
+}
